skl2onnx/operator_converters/multilayer_perceptron.py

```diff
@@ -110,9 +110,6 @@ def convert_sklearn_mlp_classifier(scope, operator, container):
     else:
         classes = np.array([s.encode('utf-8') for s in classes])
 
-    container.add_initializer(classes_name, class_type,
```

Comment:
You may also want to move the declaration on line 98.

Reply:
done